modules/ts-capi-cli.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ts-capi-cli_{context}"]
```

Comment:
this ID means nothing to me when I look at the title, but IDK If this is linked somewhere, so hesitate to recommend changing it